modules/log-collector-http-server.adoc

```diff
@@ -67,7 +67,7 @@ spec:
       receiver:
         type: http # <2>
         http:
-          format: kubeAPIAudit # <3>
+          format: kubeApiAudit # <3>
```

Comment:
The API version of this CR is `apiVersion: logging.openshift.io/v1`, is this correct?